lib/apiservers/engine/backends/system.go

```diff
@@ -329,7 +340,9 @@ func (s *SystemBackend) UnsubscribeFromEvents(listener chan interface{}) {
 
 // AuthenticateToRegistry handles docker logins
 func (s *SystemBackend) AuthenticateToRegistry(ctx context.Context, authConfig *types.AuthConfig) (string, string, error) {
-	defer trace.End(trace.Begin(""))
+	op := trace.NewOperation(context.Background(), "AuthenticateToRegistry")
+	defer trace.End(trace.Begin("", op))
+	op.Auditf("AuthenticateToRegistry")
```

Comment:
Slight deviation from our existing approach, but let's use the following here:
`op.Auditf("AuthenticateToRegistry: %s", authConfig.ServerAddress)`